app/services/find/cycle_timetable.rb

```diff
@@ -25,33 +25,30 @@ class CycleTimetable
 
         first_deadline_banner: Time.zone.local(2023, 8, 1, 9), # 5 weeks before Apply 1 deadline
         apply_1_deadline: Time.zone.local(2023, 9, 5, 18), # First Tuesday of September
-        apply_2_deadline: Time.zone.local(2023, 9, 19, 18), # 2 weeks after Apply 1 deadline
+        apply_deadline: Time.zone.local(2023, 9, 19, 18), # 2 weeks after Apply 1 deadline
         find_closes: Time.zone.local(2023, 10, 2, 23, 59, 59) # The evening before Find opens in the new cycle
       },
       2024 => {
         find_opens: Time.zone.local(2023, 10, 3, 9), # First Tuesday of October
         apply_opens: Time.zone.local(2023, 10, 10, 9), # Second Tuesday of October
-        first_deadline_banner: Time.zone.local(2024, 7, 30, 9), # 5 weeks before Apply 1 deadline
-        apply_1_deadline: Time.zone.local(2024, 9, 3, 18), # First Tuesday of September
-        apply_2_deadline: Time.zone.local(2024, 9, 17, 18), # 2 weeks after Apply 1 deadline
+        first_deadline_banner: Time.zone.local(2024, 7, 30, 9),
+        apply_deadline: Time.zone.local(2024, 9, 17, 18),
         find_closes: Time.zone.local(2024, 9, 30, 23, 59, 59) # The evening before Find opens in the new cycle
       },
       2025 => {
-        find_opens: Time.zone.local(2024, 10, 1, 9),
-        apply_opens: Time.zone.local(2024, 10, 8, 9),
-        first_deadline_banner: Time.zone.local(2025, 7, 12, 9),
-        apply_1_deadline: Time.zone.local(2025, 9, 7, 18),
-        apply_2_deadline: Time.zone.local(2025, 9, 21, 18),
-        find_closes: Time.zone.local(2025, 10, 4, 23, 59, 59)
+        find_opens: Time.zone.local(2024, 10, 1, 9), # CONFIRMED
+        apply_opens: Time.zone.local(2024, 10, 8, 9), # CONFIRMED
+        first_deadline_banner: Time.zone.local(2025, 7, 12, 9), # TBC
```

Comment:
Our deadline banner is being shown on 1 July at 6pm
```suggestion
        first_deadline_banner: Time.zone.local(2025, 7, 1, 9), # TBC
```